branca/colormap.py

```diff
@@ -70,24 +70,27 @@ class ColorMap(MacroElement):
         The right bound of the color scale.
     caption: str
         A caption to draw with the colormap.
+    max_labels : int
```

Comment:
Please also add the default value to all the docstring entries:

```suggestion
    max_labels : int, default 10
```